apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -474,9 +519,6 @@ export const ApplicantDashboardPageApi = () => {
   );
 };
 
-export const Component = () => (
-  <RequireAuth roles={[ROLE_NAME.Applicant]}>
-    <ApplicantDashboardPageApi />
-  </RequireAuth>
-);
+export default Component;
+
```

Comment:
No more need for `RequireAuth`, we can handle all that in the loader now (see: `guardByRoles`)